velero/schedule/zen5-br-scripts-cm.yaml

```diff
@@ -355,12 +353,35 @@ data:
         else
             info "Broker Secret obtained in namespace $ZEN_NAMESPACE"
         fi
-
-        extension_check=$(oc get zenextension -n $ZEN_NAMESPACE -o name | grep -v "common-web-ui-zen-extension\|zen-watchdog-frontdoor-extension" || echo "empty")
-        if [[ $extension_check != "empty" ]]; then
-            extensions=$(oc get zenextension -n $ZEN_NAMESPACE -o name | grep -v "common-web-ui-zen-extension\|zen-watchdog-frontdoor-extension" | xargs -L 1 oc get -n $ZEN_NAMESPACE -o jsonpath={.spec.extensions} | grep extension_name | awk '{print $2}' | tr '\"\n' ' ' | tr ',' ' ')
-            extensions=$(echo $extensions | tr " " ",") #clean up the output for later use
-            
+        if [[ $BACKUP == "true" ]]; then
+            extension_check=$(oc get zenextension -n $ZEN_NAMESPACE -o name | grep -v "common-web-ui-zen-extension\|zen-watchdog-frontdoor-extension" || echo "empty")
+            if [[ $extension_check != "empty" ]]; then
+                extensions=""
+                for ext in $extension_check
+                do
+                    ext_name=$(oc get $ext -n $ZEN_NAMESPACE -o jsonpath={.spec.extensions} | grep extension_name | awk '{print $2}' | tr '\"\n' ' ' | tr "," " ")
+                    if [[ $extensions == "" ]]; then
+                        extensions="$ext_name,"
+                    else
+                        extensions="$extensions,$ext_name"
+                    fi
+                done
+                
+                extensions=$(echo $extensions | tr "," " ")
+                echo $extensions
+                
+                cat <<< $extensions > /tmp/extensions.txt
```

Comment:
@bluzarraga , Hi Ben, the `/tmp/` folder is not in the backup scope, I guess you need persist this file at `/zen5/zen-backup` or its sub folder

Reply:
Oh, I see there is a new "emptyDir" mounted at the backup pod, but I'm wondering why you do not use the `/zen5/zen-backup`. The emptyDir is usually for temporary persistency request like cache. Seems like it's any pattern to be considered at the backup. Just my gut feeling.

Reply:
changed the directory to /zen5/extensions so it is in the same parent directory as the rest of the backup